### PR TITLE
fixed #11500 Add tip about filter proxy

### LINKED ISSUE #11500: Add tip about filter proxy to Docker installation Guides
Add information for getting around key errors when installing Docker on Ubuntu and Debian 
-->
If your company is behind a filtering proxy, you may find the `apt-key` command fails for the Docker repo during installation. To work around this, add the key directly using the following:
 
`wget -qO- https://get.docker.com/gpg  | sudo apt-key add -`

### PR DESCRIPTION
**Pull request by xiaods**
Add tip about filter proxy to Docker installation Guides

Signed-off-by: Deshi Xiao xiaods@gmail.com

### CODE REVIEW

File: docs/sources/installation/debian.md [hunk 1]
```diff
@@ -64,6 +64,9 @@ which is officially supported by Docker.
 3. Install Docker using the get.docker.com script:
  
     `curl -sSL https://get.docker.com/ | sh`
+    > **tips** If your company is behind a filtering proxy, you may find the apt-key command fails for the Docker repo during isntalllation. To work around this, add the key directly using the following:
```
Comment (moxiegirl) [inline docs/sources/installation/debian.md]: Hi thanks for the PR.  Couple of things:
- wrap lines at 80 characters please
- we prefer notes over tips
- there was a spelling error "installation"

https://gist.github.com/moxiegirl/6b71ed4941952d689383

-->Fix:

>**Note**: If your company is behind a filtering proxy, you may find the `apt-key`
>command fails for the Docker repo during installation. To work around this,
>add the key directly using the following:
>
> 	    $ wget -qO- https://get.docker.com/gpg | sudo apt-key add -

File: docs/sources/installation/ubuntulinux.md [hunk 2]
```diff
@@ -114,6 +114,9 @@ install Docker using the following:
 
 	 The system prompts you for your `sudo` password. Then, it downloads and
 	 installs Docker and its dependencies.
+	 > **tips** If your company is behind a filtering proxy, you may find the apt-key command fails for the Docker repo during isntalllation. To work around this, add the key directly using the following:
```
Comment (moxiegirl) [inline docs/sources/installation/ubuntulinux.md]: Same comment as above.

**Comment by xiaods** [comment]
@moxiegirl done. refresh the PR.

**Comment by moxiegirl** [comment]
@xiaods  Thank you! Very quick.  LGTM

@jamtur01 @fredlf @thaJeztah

**Comment by fredlf** [comment]
LGTM

File: docs/sources/installation/debian.md [hunk 3]
```diff
@@ -62,9 +62,16 @@ which is officially supported by Docker.
 2. Restart your system. This is necessary for Debian to use your new kernel.
 
 3. Install Docker using the get.docker.com script:
- 
+
     `curl -sSL https://get.docker.com/ | sh`
 
+>**Note**: If your company is behind a filtering proxy, you may find the
```
Comment (thaJeztah) [inline docs/sources/installation/debian.md]: should it be; you may find **that** the..?

Comment (xiaods) [inline docs/sources/installation/debian.md]: @thaJeztah udpate it right now. hold on

**Comment by thaJeztah** [comment]
Added one note (but I'm not sure it's correct); LGTM

You've tested the instructions, @xiaouds?

**Comment by xiaods** [comment]
@thaJeztah  yes, i have tested the script, it's ok from my environment.

**Comment by thaJeztah** [comment]
Thanks! LGTM

ping @moxiegirl

**Pull request merged**